src/core/foundations/src/utils/css.ts

```diff
@@ -0,0 +1,2 @@
+export const appearance =
+	'(appearance: none) or (-webkit-appearance: none) or (-moz-appearance: none)';
```

Comment:
Following #981, this should probably be in `packages/@guardian/source-foundations/src/utils` with a symlink in `src/core/foundations/src/utils`

Reply:
`packages/@guardian/source-foundations/src/utils/supports-queries.ts`?